basic.obo

```diff
@@ -215519,10 +216730,8 @@ created_by: WD
 
 [Term]
 id: UBERON:4300187
-name: buccopharyngeal complex
-def: "Anatomical cluster that is located in the oral cavity and consists of modifications to the skin on the roof of the mouth into three primary skin folds. In addition to these longitudinally oriented folds, secondary folds are present in some taxa.[PHENOSCAPE:WD]" [https://doi.org/10.5479/si.00810282.471]
-is_a: UBERON:0000477 ! anatomical cluster
-relationship: part_of UBERON:0000165 ! mouth
+name: obsolete buccopharyngeal complex
+is_obsolete: true
 created_by: WD
```

Comment:
Newly obsolete term missing consider or replaced_by tags

Reply:
Is there any discussion about obsoleting this term? I searched issues and didn't find any. We have Phenoscape annotations.

Reply:
I couldn't find an issue or PR either.  Maybe safest to just reverse.  @cmungall any idea where this might have come from?

Reply:
@dosumis I found it: https://github.com/obophenotype/uberon-phenoscape-ext/commit/afbc91c8ba4e9f2574719fba1614348b9195c2a9

This was done by @wdahdul so it must be fine from a Phenoscape perspective. :-)

Reply:
Maintaining some of the ontology is a separate repository will probably continue to be a source of confusion. We should aim to merge into one.